app/frontend/components/pages/delegations/currentDelegationPage.js

```diff
@@ -0,0 +1,102 @@
+const {h} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../../actions')
+
+const CopyOnClick = require('../../common/copyOnClick')
+
+const CurrentDelegationPage = ({currentStakepool, undelegateStakepool}) => {
+  return h(
+    'div',
+    {class: 'current-delegation card'},
+    h('h2', {class: 'card-title'}, 'Current delegation'),
+    currentStakepool.address === ''
+      ? h(
+        'a',
+        {
+          class: 'undelegated',
+        },
+        'Your Ada is undelegated. Delegate now!'
+      )
+      : h(
+        'div',
+        {class: 'stakepool-info'},
+        h(
+          'div',
+          {class: 'stakepool-name-wrapper'},
+          h('p', {class: 'stakepool-name'}, `${currentStakepool.name}`),
```

Comment:
Since we're doing camelCase, please change "currentStakepool" to "currentStakePool" since it's not a single word. Same goes for every occurence of "stakepool" -> "stakePool" in the code.

Reply:
I had my doubts indeed. Fixed.